testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java

```diff
@@ -300,6 +300,16 @@ public Pattern[] get(boolean inContainer) {
                     p.add(Pattern.compile(".*sun.misc.Unsafe::objectFieldOffset has been called by org.jboss.threads.JBossExecutors.*"));
                     p.add(Pattern.compile(".*java.lang.System::load has been called by com.sun.jna.Native in.*net/java/dev/jna.*"));
                 }
+                if (QUARKUS_VERSION.compareTo(new QuarkusVersion("3.28.1")) >= 0 || QUARKUS_VERSION.isSnapshot()) {
+                    p.add(Pattern.compile(".*JDBC fetch size: undefined/unknown.*"));
+                    p.add(Pattern.compile(".*Pool: undefined/unknown.*"));
+                    p.add(Pattern.compile(".*catalog/schema: unknown/unknown.*"));
+                }
+            }
+            if ((UsedVersion.getVersion(inContainer).compareTo(Version.create(25, 0, 0)) >= 0)) {
+                // https://github.com/quarkusio/quarkus/issues/47769
+                p.add(Pattern.compile(".*to use the thread-local-reset capability on Java 24 or later.*"));
+                p.add(Pattern.compile(".*Could not initialize class org.jboss.threads.JDKSpecific.*"));
             }
```

Comment:
Where do we see those? A better fix would be to run `native-image` with `-J--add-opens=java.base/java.lang=ALL-UNNAMED` since we know it's needed for a newly generated quarkus app. This avoids not seeing them when we actually don't expect it.

Reply:
>  The current approach in this patch disables the warning for all apps in the TS when run on Mandrel 25+.

Not true, it affects only this app: QUARKUS_MP_ORM_DBS_AWT

So it's really a matter of either adding the `-J--add-opens` to the command line or whitelisting it here.

I am O.K. with adding it to the app's command line though, so I'll refactor it...

Reply:
> > The current approach in this patch disables the warning for all apps in the TS when run on Mandrel 25+.
> 
> Not true, it affects only this app: QUARKUS_MP_ORM_DBS_AWT

Ah, right. My bad.